IdentityServer/v7/docs/content/overview/security.md

```diff
@@ -67,21 +67,61 @@ Duende IdentityServer is a [certified](https://openid.net/certification/) implem
 ### Package Signing
 
 NuGet packages published by Duende are cryptographically signed to ensure their
-authenticity and integrity. Our certificate is signed by Sectigo, which is a widely
+authenticity and integrity. Our certificate is signed by DigiCert, which is a widely
 trusted certificate authority and installed by default in most environments. This means
 that in many circumstances, the NuGet tools can validate our packages' signatures
 automatically.
 
-However, some environments (notably the dotnet sdk docker image which is
-sometimes used in build pipelines) do not trust the Sectigo certificate. In that case, it might be necessary to add the
-Sectigo root certificate to NuGet's code signing certificate bundle. Sectigo's root
-certificate is available from Sectigo
-[here](http://crt.sectigo.com/SectigoPublicCodeSigningRootR46.p7c).
+However, some environments (notably the dotnet sdk docker image which is  sometimes used in
+build pipelines) do not trust the certificate. In that case, it might be necessary to add the
+root certificate to NuGet's code signing certificate bundle.
+
+* Packages released after January 1, 2025 (IdentityServer 7.1+): Use DigiCert's [root certificate](https://cacerts.digicert.com/DigiCertTrustedG4CodeSigningRSA4096SHA2562021CA1.crt.pem) (PEM).
+* Packages released before January 1, 2025: Use Sectigo's [root certificate](http://crt.sectigo.com/SectigoPublicCodeSigningRootR46.p7c) (P7C).
+
+#### Trusting the DigiCert certificate
+
+Here is an example of how to configure NuGet to trust the DigiCert root CA on the dotnet sdk docker image.
+This applies for Duende packages released **after** January 1, 2025, such as IdentityServer 7.1 and newer versions.
+
+First, get the DigiCert certificate:
+
+```sh
+wget https://cacerts.digicert.com/DigiCertTrustedG4CodeSigningRSA4096SHA2562021CA1.crt.pem
```

Comment:
Maybe we should mention that `wget` is available on Chocolatey and Homebrew for the less CLI savvy.

Reply:
Actually, I see there are more command line utilities below; mentioning them as prerequisites is a good idea.

Reply:
Updated